README.rst

```diff
@@ -119,7 +119,8 @@ After you have setup the local installation of Batavia, you can now run Python i
 
 $ cd testserver
 $ pip install -r requirements.txt
-$ ./manage.py runserver
+
```

Comment:
Sorry to be nitpicky here - but a stray newline has slipped in here.

Reply:
I'm going to merge this and then fix the newline after.